Sorry, it's so late and not completely. I will definitely finish it ...

KatovichAlexander opened the pull request.

KatovichAlexander committed:
Sorry, it's so late and not completely. I will definitely finish it ...

GlebLitvinov reviewed (commented):
okay, I will wait for the  3rd task. First 2 tasks are good. (the only thing - work with int number , not with int "a" in operation method

KatovichAlexander committed:
The third task + corrected the first two.

GlebLitvinov commented on Lesson3/src/Homework.java:
```diff
@@ -0,0 +1,194 @@
+
+        Scanner scanner1 = new Scanner(System.in);
+        System.out.print("Input width: ");
+        int width = scanner1.nextInt();
```
you can read width using scanner, creating another one is not necessary

GlebLitvinov commented on Lesson3/src/Homework.java:
```diff
@@ -0,0 +1,194 @@
+
+        for (int[] row : twoDArray) {
+            for (int element : row) {
+                if (element % 3 == 0) {
```
this check is correct but you also need to do a check that element can be divided by 7, and in this case print "-"

GlebLitvinov reviewed (changes requested):
overall is good, fix minor things that I mentioned and we are good to go

KatovichAlexander committed:
you can read width using scanner, creating another one is not necessary - fix.
this check is correct but you also need to do a check that element can be divided by 7, and in this case print "-" - fix

GlebLitvinov commented:
Good job
